config/puma.rb

```diff
@@ -21,7 +21,7 @@
 # Workers do not work on JRuby or Windows (both of which do not support
 # processes).
 #
-workers ENV.fetch("WEB_CONCURRENCY") { 2 }
+# workers ENV.fetch("WEB_CONCURRENCY") { 2 }
```

Comment:
I think you mistakenly committed this file, kindly remove it

Reply:
> I think you mistakenly committed this file, kindly remove it

I commented this line, because the project wasn't working for me. So, I searched on net and found this solution to comment it out. So, I'll again just uncomment it and push it. Is that good?

Reply:
Yaa I have also faced this problem while setup on windows.
You can uncomment it and push it again.